data/src/test/java/org/apache/iceberg/data/TestDataFileIndexStatsFilters.java

```diff
@@ -94,24 +93,23 @@ public void createTableAndData() throws IOException {
             .filter(rec -> rec.getField("category").equals("even"))
             .collect(Collectors.toList());
 
-    this.dataFile = FileHelpers.writeDataFile(table, Files.localOutput(temp.newFile()), records);
```

Comment:
why is this being removed? I think this should be `this.dataFile = FileHelpers.writeDataFile(table, Files.localOutput(File.createTempFile("junit", null, temp.toFile())), records);`